notebooks/MAST/spectraldb_and_specviz/spectraldb_and_specviz.ipynb

```diff
@@ -0,0 +1,919 @@
+{
```

Comment:
<p>In some places you reference the API with the database itself, "Spectral DB API". Since most users won't know about the database itself, but access it via the API only, we might want to instead refer to the API as the Spectral Pixel Search API, which is the public-facing service to search the SpectralDB. We can just make a brief comment that the SpectralDB is a new database holding the pixel-level information for spectroscopic data products.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/16/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>Replaced 3rd sentence with: "The SpectralDB is a new database holding the pixel-level information for spectroscopic dataproducts. Using the Spectral Pixel Search API, we can search the SpectralDB by pixel-level information..."</p>

Reply:
<p>And "The URLs for searching with the Spectral DB API..." with "The URLs for searching with the Spectral Pixel Search API ..." will also look for other places where I do this.</p>